<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HiddenCat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openBtn">
        <property name="text">
         <string>Open Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aboutBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>About HiddenCat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decodeBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Decode Image...</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Save Encoded</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Zoom:  -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="zoomSlider">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>800</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="zoomLbl">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>100%</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="noShameChckBx">
            <property name="text">
             <string>Show Encoded Area</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="modImageView">
          <property name="styleSheet">
           <string notr="true">QGraphicsView { background-color: gray; }</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="interactive">
           <bool>false</bool>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Image with Hidden Message</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Bits used per pixel: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="bitsCount">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="bitsUsedCount">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>24</number>
              </property>
              <property name="pageStep">
               <number>3</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::NoTicks</enum>
              </property>
              <property name="tickInterval">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="charCounter">
              <property name="text">
               <string>Message Length: 0</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="maxCharsCounter">
              <property name="text">
               <string>Maximum: - </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="message">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Message to Encode</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bitsUsedCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitsCount</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
